gax-grpc/src/main/java/com/google/api/gax/grpc/OperationCallable.java

```diff
@@ -27,148 +27,112 @@
+      RequestT request, ApiCallContext context) {
+    ApiCallContext newCallContext = context;
+    if (callContextDecorator != null) {
+      newCallContext = callContextDecorator.decorate(context);
     }
-    return futureCall(initialCallable.futureCall(request, context));
-  }
-
-  OperationFuture<ResponseT, MetadataT> futureCall(ApiFuture<Operation> initialFuture) {
-    RetryingCallable<RequestT, Operation> callable =
-        new RetryingCallable<>(
-            new OperationCheckingCallable<RequestT>(
-                operationsClient.getOperationCallable(), initialFuture),
-            executor);
-
-    RetryingFuture<Operation> pollingFuture = callable.futureCall(null, null);
-    return new OperationFuture<>(
-        pollingFuture, initialFuture, responseTransformer, metadataTransformer);
+    return callable.futureCall(request, newCallContext);
   }
 
   /**
-   * Same as {@link #futureCall(Object, CallContext)}, with null {@link io.grpc.Channel} and default
-   * {@link io.grpc.CallOptions}.
+   * Same as {@link #futureCall(Object, ApiCallContext)}, with a null context.
    *
-   * @param request The request to initiate the operation.
-   * @return {@link OperationFuture} for the call result
+   * @param request request
+   * @return {@link ApiFuture} for the call result
    */
-  public OperationFuture<ResponseT, MetadataT> futureCall(RequestT request) {
-    return futureCall(request, CallContext.createDefault().withChannel(clientContext.getChannel()));
+  public OperationFuture<ResponseT, MetadataT, OperationT> futureCall(RequestT request) {
+    return futureCall(request, null);
   }
 
   /**
-   * Initiates an operation and polls for the final result. If the {@link io.grpc.Channel}
-   * encapsulated in the given {@link com.google.api.gax.grpc.CallContext} is null, a channel must
-   * have already been bound at construction time.
+   * Perform a call synchronously.
    *
-   * @param request The request to initiate the operation.
-   * @param context {@link com.google.api.gax.grpc.CallContext} to make the call with
+   * @param request The request to send to the service.
+   * @param context {@link ApiCallContext} to make the call with
    * @return the call result
    * @throws ApiException if there is any bad status in the response.
    * @throws RuntimeException if there is any other exception unrelated to bad status.
    */
-  public ResponseT call(RequestT request, CallContext context) {
+  public ResponseT call(RequestT request, ApiCallContext context) {
     return ApiExceptions.callAndTranslateApiException(futureCall(request, context));
   }
 
   /**
-   * Same as {@link #call(Object, CallContext)}, with null {@link io.grpc.Channel} and default
-   * {@link io.grpc.CallOptions}.
+   * Same as {@link #call(Object, ApiCallContext)}, with a null context.
    *
-   * @param request The request to initiate the operation.
+   * @param request The request to send to the service.
```

Comment:
I believe the old documentation was more informative (we should tell user somehow that the LRO does not consist from one single request, but with multiple polling request after that, and those are not the request he provides initially).